vim_menutrans_helper.py

```diff
@@ -0,0 +1,228 @@
+                continue
+
+            if word_list[0] in sil_commands:
+                index += 1
+            # Blocks needs to be translated without being split by '.'
+            if index < len(word_list) and word_list[index] in menu_commands:
+                to_be_translated_blocks = ""
+                index += 1
+                while index < len(word_list):
+                    if (word_list[index] == '<silent>' or 
+                            word_list[index] == '<script>' or 
+                            (word_list[index][0].isdigit() and 
+                            not word_list[index - 1][0].isdigit())):
+                        index += 1
+                    else:
+                        # Menu_item with "ToolBar" has no following words to be 
+                        # translated
+                        if not ("ToolBar." in word_list[index]):
+                            to_be_translated_blocks = word_list[index]
+                        break
+
+                # Remove the redundant digit added at the head and tail of the 
+                # mid_part
+                if (len(to_be_translated_blocks) > 0 and 
+                        to_be_translated_blocks[0] == '5'):
+                    to_be_translated_blocks = to_be_translated_blocks[1:]
+                if (len(to_be_translated_blocks) > 0 and 
+                        to_be_translated_blocks[-1] == '5'):
+                     to_be_translated_blocks = to_be_translated_blocks[:-1]
+
+                # Menu_item with "ToolBar" constructed a tobe_translated_blocks 
+                # with length equals zero
+                if len(to_be_translated_blocks) == 0:
+                    continue
+
+                # Untranslated word string split by "." but with escaped 
+                # literal dot character"\." together
+                to_be_translated_words = []
+                for match in re.finditer(r"(?:\\.|[^.])+", 
+                        to_be_translated_blocks):
+                    word = to_be_translated_blocks[match.start(): match.end()]
+                    if not (len(word) != 0 and word[0] == '-'):
+                        to_be_translated_words.append(word)
+
+                # Conver tobe_translated_words list to string
+                for word in to_be_translated_words:
+                    if len(word) > 0:
+                        untranslated_dict[word.lower()] = (line_number, 
+                                untranslation_file, word)
+            elif word_list[0] in toolbar_commands:
+                untranslated_dict[word_list[1].lower()] = (line_number, 
+                        untranslation_file, word_list[1])
+            elif word_list[0] == "let" and (len(word_list) > 1 and 
+                    word_list[1].startswith("g:menutrans_")):
+                untranslated_dict[word_list[1].lower()] = (line_number, 
+                        untranslation_file, word_list[1])
+
+    # PopUp doesn't need to be translated
```

Comment:
This comment is not quite clear. "PopUp" is used to define popup menus. The sub menu items under "PopUp" need to be translates. But "PopUp" itself is just a placeholder string. It doesn't need translation.